src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubTriggerAttributeBindingProvider.cs

```diff
@@ -62,7 +62,7 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
             Func<ListenerFactoryContext, bool, Task<IListener>> createListener =
              (factoryContext, singleDispatch) =>
              {
-                 IListener listener = new EventHubListener(factoryContext.Executor, eventHostListener, options, singleDispatch);
+                 IListener listener = new EventHubListener(factoryContext.Executor, eventHostListener, options, singleDispatch, _eventHubConfig.BatchCheckpointFrequency);
```

Comment:
Reasonable - we source eventHostListener, options, and the checkpoint frequency from the config. A lot of this is testability, but as long as we're only pulling poco properties off the config it should be no big deal.